pkg/operator/ceph/file/filesystem.go

```diff
@@ -140,6 +141,25 @@ func newFS(name, namespace string) *Filesystem {
 	}
 }
 
+// SetPoolSize method sets the sizes for MetadataPool and dataPool
+func SetPoolSize(context *clusterd.Context, clusterName, fsName string, dataPool []*model.Pool, metadataPool *model.Pool) error {
+	// generating the metadata pool's name
+	metadataPool = generateMetadataPoolName(fsName, metadataPool)
+	err := client.CreatePoolWithProfile(context, clusterName, *metadataPool, appName)
+	if err != nil {
+		return errors.Wrapf(err, "failed to update metadata pool %q", *&metadataPool.Name)
```

Comment:
`*&metadataPool.Name` ???

Reply:
I was using this to point the Pool's name, strange that after reading, it is not reaching to Pool's name anymore.